src/app/first/first.component.html

```diff
@@ -1,10 +1,32 @@
 <div class="padded">
-  <h1>FirstComponent</h1>
 
-  <div class="btn-group" role="group" aria-label="Basic example">
-    <button type="button" class="btn btn-secondary">first menu</button>
-    <button type="button" class="btn btn-secondary">second menu</button>
-    <button type="button" class="btn btn-secondary">third menu</button>
-    <button type="button" class="btn btn-secondary">fourth menu</button>
+  <div class="container-1">
+    <h1 class="vert-stack"> FirstComponent</h1>
+
+    <div class="btn-group" role="group" aria-label="Basic example">
+      <button type="button" class="btn btn-secondary">first menu</button>
+      <button type="button" class="btn btn-secondary">second menu</button>
+      <button type="button" class="btn btn-secondary">third menu</button>
+      <button type="button" class="btn btn-secondary">fourth menu</button>
+    </div>
+  </div>
+  <div class="container-1">
+    <h1>Flexbox?</h1>
```

Comment:
flexbox is great... I refer to this all the time [css-tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

Another great layout helper is the Grid layout from [bootstrap/coreui](https://coreui.io/docs/2.1/layout/grid/)